main/create_simulated_body_model.py

```
@@ -0,0 +1,1085 @@
+            'Save Image',
+            'simulated_model.png',
+            'PNG Images (*.png)',
+        )
+        if filename:
+            try:
+                from PIL import Image
+                img_uint8 = (
+                    np.clip(self._current_image, 0.0, 1.0) * 255
+                ).astype(np.uint8)
+                Image.fromarray(img_uint8, mode='L').save(filename)
+            except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)